docs/api/content-tracing.md

```diff
@@ -121,22 +125,23 @@ Get the current monitoring traced data.
 
 Child processes typically are caching trace data and only rarely flush and send
 trace data back to the main process. That is because it may be an expensive
-operation to send the trace data over IPC, and we would like to avoid unneeded
-runtime overhead of tracing. So, to end tracing, we must asynchronously ask all
-child processes to flush any pending trace data.
+operation to send the trace data over IPC and we would like to avoid unneeded
+runtime overhead from tracing. So, to end tracing, we must asynchronously ask
+all child processes to flush any pending trace data.
 
-Once all child processes have acked to the `captureMonitoringSnapshot` request,
-the `callback` will be invoked with a file that contains the traced data.
+Once all child processes have acked to the `captureMonitoringSnapshot` request
+the `callback` will be called with a file that contains the traced data.
 
 
-## tracing.getTraceBufferUsage(callback)
+### `tracing.getTraceBufferUsage(callback)`
 
 * `callback` Function
 
-Get the maximum across processes of trace buffer percent full state. When the
-TraceBufferUsage value is determined, the `callback` is called.
+Get the maximum usage across processes of trace buffer as a percentage of the
+full state. When the TraceBufferUsage value is determined the `callback` is
+called.
```

Comment:
Looks good to me, I just copied it from [Chromium's comments](https://code.google.com/p/chromium/codesearch#chromium/src/content/public/browser/tracing_controller.h&q=getTraceBufferUsage&sq=package:chromium&type=cs&l=199).